cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -575,6 +576,15 @@ export function activateObject(activatedStateID: number | null, activatedAttribu
     };
 }
 
+export function updateTabContentHeight(tabContentHeight: number): AnyAction {
+    return {
+        type: AnnotationActionTypes.UPDATE_TAB_CONTENT_HEIGHT,
+        payload: {
+            tabContentHeight,
+        },
+    };
+}
+
```

Comment:
Code related with tab height was removed in develop

Reply:
Removed